packages/host/app/components/operator-mode/card-adoption-chain.gts

```diff
@@ -22,16 +24,60 @@ export default class CardAdoptionChain extends Component<Signature> {
       .card-adoption-chain {
         background-color: var(--boxel-200);
       }
+      .inherits-from {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        padding: var(--boxel-sp) 0;
+
+        position: relative;
+      }
+
+      .inherits-icon {
+        height: 25px;
+      }
+
+      .line1 {
+        width: 50%;
+        height: 2px;
+        background-color: var(--boxel-dark);
+        position: absolute;
+        top: 50%;
+        left: 0;
+        transform: translateX(-50%);
+      }
+
+      .line2 {
+        width: 50%;
+        height: 2px;
+        background-color: var(--boxel-dark);
+        position: absolute;
+        top: 50%;
+        right: 0;
+        transform: translateX(50%);
+      }
     </style>
 
     <div class='card-adoption-chain'>
-      {{#each @cardInheritanceChain as |data|}}
+      {{#each @cardInheritanceChain as |data index|}}
         <CardSchemaEditor
           @card={{data.card}}
           @cardType={{data.cardType}}
           @file={{@file}}
           @moduleSyntax={{this.moduleSyntax}}
         />
+        {{#unless (this.isLastIndex index)}}
```

Comment:
Can't this be solved with css? `last-child`?

Reply:
updated [75d6021](https://github.com/cardstack/boxel/pull/707/commits/75d6021c4bec46b79afd7a967d1c58d3066281d2)